### ISSUE TITLE
Contributed icons get added twice into document style

### ISSUE THREAD

**Issue by tsmaeder** [eclipse-theia/theia#13151]
### Bug Description:
When a plugin contributes an icon, the icon gets added twice to the document stylesheet: once into a style element with id `contributedIconsStyles` and once into a style element with class `monaco-colors`. Depending on unclear factors, one or the other definition will be used. 

The one that does _not_ seem to work comes from the `monaco-colors` element and it looks like this:

```
.codicon-gitlens-gitlens:before {
    content: '\f113';
    font-family: 'eamodio.gitlens/file:/c%3A/Users/thomas/AppData/Local/Temp/vscode-unpacked/eamodio.gitlens-14.5.1.vsix/extension/dist/glicons.woff2';
}
```

When this happens, I also have this error message in the browser console: 
```
Refused to apply inline style because it violates the following Content Security Policy directive: "style-src <URL> 'nonce-nhnh5TrVIuLLegOmT7xetg==' 'unsafe-hashes' 'sha256-mUrDqrM3P6dEDkmjra3Mfhlzj64svqv7cS2Py/MWO6g=' 'sha256-Rmq/hkXE1x2LI3jhwuDLafAomGzcIUAF/Q8OQgLnIO8='". Either the 'unsafe-inline' keyword, a hash ('sha256-47DEQpj8HBSa+/TImW+5JCeuQeRkm5NMpJWZG3hSuFU='), or a nonce ('nonce-...') is required to enable inline execut

```

However, I am not sure whether the these occurrences are related

### Steps to Reproduce:

1. Set up Theia to use the git VS Code extension instead of the Theia git package
2. Install gitlens
3. Delete the electron data area (`<home>\AppData\Roaming\TheiaElectronExample` on Windows)
4. Start Theia electron
5. Open a workspace
6. Do "Open Recent Workspace"
7. Observe: sometimes, the icons for the the gitlens views in the left bar are broken
8. As an added bonus, the "Gitlens" view that lives in the lower right area cannot be created. 

This seems to be timing-dependent, so your mileage may vary.

### Additional Information

- Operating System:
- Theia Version:

**Comment by tsmaeder** [comment]
About the error message: note that VS Code has a conten-policy and Theia does not:
```
<meta http-equiv="Content-Security-Policy" content="
				default-src
					'none'
				;
				img-src
					'self'
					data:
					blob:
					vscode-remote-resource:
					vscode-managed-remote-resource:
					https:
				;
				media-src
					'self'
				;
				frame-src
					'self'
					vscode-webview:
				;
				script-src
					'self'
					'unsafe-eval'
					blob:
				;
				style-src
					'self'
					'unsafe-inline'
				;
				connect-src
					'self'
					https:
					ws:
				;
				font-src
					'self'
					vscode-remote-resource:
					vscode-managed-remote-resource:
				;
				require-trusted-types-for
					'script'
				;
				trusted-types
					amdLoader
					cellRendererEditorText
					defaultWorkerFactory
					diffEditorWidget
					diffReview
					domLineBreaksComputer
					dompurify
					editorGhostText
					editorViewLayer
					notebookRenderer
					stickyScrollViewLayer
					tokenizeToString
				;
		">
```

**Comment by tsmaeder** [comment]
This has been addressed as part of the monaco upgrade to 1.83.1: https://github.com/eclipse-theia/theia/pull/13217/files#diff-3cae0aeb56ecaaf91597d965b8a302ea4ea3f9dfdb4eb037b6a4213f9de80abc
